### PR TITLE
Translate follow up for document update (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Follow up #2911, #2915

Please review this 🙏

### CODE REVIEW

File: ko/documentation/installation/index.md [hunk 1]
```diff
@@ -316,6 +316,13 @@ rbenv 자체는 Ruby 설치를 지원하지 않습니다만, [ruby-build](#ruby-
 유명한 플러그인에서 Ruby를 설치할 수 있습니다.
 rbenv, ruby-build 모두 macOS, Linux나 다른 UNIX-계열 운영체제에서 사용가능합니다.
 
+### rbenv for windows
```
Comment (yous) [inline ko/documentation/installation/index.md:319]:
```suggestion
### rbenv for Windows
```

File: ko/documentation/installation/index.md [hunk 2]
```diff
@@ -316,6 +316,13 @@ rbenv 자체는 Ruby 설치를 지원하지 않습니다만, [ruby-build](#ruby-
...
+{: #rbenv-for-windows}
+
+[rbenv for Windows][rbenv-for-windows]는 여러 종류의 Ruby on Windows를 설치하고
```
Comment (yous) [inline ko/documentation/installation/index.md:322]:
```suggestion
[rbenv for Windows][rbenv-for-windows]는 Windows에서 여러 종류의 Ruby를 설치하고
```

File: ko/documentation/installation/index.md [hunk 3]
```diff
@@ -316,6 +316,13 @@ rbenv 자체는 Ruby 설치를 지원하지 않습니다만, [ruby-build](#ruby-
...
+관리할 수 있도록 합니다. 이는 PowerShell로 작성되었으므로 Windows에서 루비를
```
Comment (yous) [inline ko/documentation/installation/index.md:323]:
```suggestion
관리할 수 있도록 합니다. 이는 PowerShell로 작성되었으므로 Windows에서 Ruby를
```

File: ko/documentation/installation/index.md [hunk 4]
```diff
@@ -316,6 +316,13 @@ rbenv 자체는 Ruby 설치를 지원하지 않습니다만, [ruby-build](#ruby-
...
+이용하는 사용자들에게 자연스러운 수단을 제공합니다. 나아가, 커맨드 라인 인터페이스는
```
Comment (yous) [inline ko/documentation/installation/index.md:324]:
```suggestion
이용하는 사용자들에게 자연스러운 수단을 제공합니다. 또한 커맨드 라인 인터페이스는
```

**Review by yous — APPROVED** [review]
Thanks!

**Comment by riseshia** [comment]
🙏

**Pull request merged**